### DISCUSSION TITLE
Wazuh Groups

### DISCUSSION THREAD

**Discussion by KompTi** [wazuh/wazuh#15445] [Q&A] [answered]
Hi everyone,

Please, I need your support...

How many labels of groups can I use in Wazuh for each agent?

Beacause, we need create groups to identify: Servers, Desktop, SO, Departmen (for example: Project - Design - Architecture) 

![image](https://user-images.githubusercontent.com/118565014/202798854-24cdc4c8-52d9-4c35-b985-549ff946127b.png)

Can I assign three or more labels? 

Thank a lot

**Comment by Selutario** [comment, marked as answer]
Hi @KompTi,

Sure, you can assign more than 3 groups. Currently the limit is **256** groups per agent, although with the arrival of Wazuh v4.4 and to avoid certain problems, a maximum number of **128** groups per agent will be introduced. You can read about it in this issue:
- #11312

Those limits should not be a problem since, as mentioned in the issue, the groups are merged according to their priority. Therefore, the first configuration group in a merged configuration file will probably be totally overridden. 

This feature (multiple groups per agent) was originally introduced in Wazuh v3.7.0. You can read more about it in the documentation:
- [Adding agents to multiple groups - 3.7.0 Release notes - 10 November 2018](https://documentation.wazuh.com/current/release-notes/release-3-7-0.html#adding-agents-to-multiple-groups)
- [Assigning multiple groups to an agent - Grouping agents](https://documentation.wazuh.com/current/user-manual/agents/grouping-agents.html#assigning-multiple-groups-to-an-agent)

**Comment by KompTi** [comment]
Thank a lot @Selutario & @jctello 

We appreciate your great help!
